BurnOutSharp/ProtectionType/SafeDisc.cs

```diff
@@ -25,6 +43,20 @@ public string CheckPortableExecutable(string file, PortableExecutable pex, bool
             if (!string.IsNullOrWhiteSpace(name) && name.Equals("SafeCast2", StringComparison.OrdinalIgnoreCase))
                 return $"SafeCast";
 
+            // Present on all "CLOKSPL.EXE" versions before SafeDisc 1.06.000. Found on Redump entries 61731 and 66004. 
+            // Only found so far on SafeDisc 1.00.025-1.01.044, but the report is currently left generic due to the generic nature of the check.
+            if (!string.IsNullOrWhiteSpace(name) && name.Equals("SafeDisc", StringComparison.OrdinalIgnoreCase))
+                return $"SafeDisc";
```

Comment:
In the comment, it notes that this shows up for a certain range. If it's that generic, it feels like this check should be later on to allow other specific checks to happen first.